Formula/got.rb

```diff
@@ -0,0 +1,27 @@
+class Got < Formula
+  desc "Version control system"
+  homepage "https://gameoftrees.org"
+  url "https://gameoftrees.org/releases/portable/got-portable-0.75.tar.gz"
+  sha256 "b35324893c9aefbd6bf49602a6c9bb9d6e20670aff3c9734fefead0205bb9cb2"
+  license "ISC"
+
+  depends_on "ncurses"
+  depends_on "openssl@3"
+  depends_on "ossp-uuid"
+
+  uses_from_macos "bison" => :build
```

Comment:
Looks like this needs to be `depends_on` as well.

Reply:
How do you see that? Asking to learn. 🙂

Reply:
That's a guess based on the CI failure here: https://github.com/Homebrew/homebrew-core/actions/runs/3062430582/jobs/4943465511#step:6:189